src/Modal/Modal.component.tsx

```diff
@@ -4,12 +4,20 @@ import * as React from 'react';
 // VENDOR
 import * as StyledReactModal from 'styled-react-modal';
 import styled, { css } from 'styled-components';
+import {
+    compose,
+    variant,
+    borderRadius as borderRadiusStyles,
```

Comment:
I aliased these to avoid shadow-naming tslint errors, due to the prop variables given defaults having the same names.

Reply:
I found this out when trying to delete them locally. Not a bad way to avoid the linter errors. We could modify the rules, but I like this better.